app/views/content/cookies.html.erb

```diff
@@ -1,140 +1,138 @@
-            Name
-          </th>
-          <th scope="col" class="govuk-table__header">
-            Purpose
-          </th>
-          <th scope="col" class="govuk-table__header">
-            Expires
-          </th>
-        </tr>
+      <tr class="govuk-table__row">
+        <th scope="col" class="govuk-table__header govuk-!-width-one-third">
+          Name
+        </th>
+        <th scope="col" class="govuk-table__header govuk-!-width-one-third">
+          Purpose
+        </th>
+        <th scope="col" class="govuk-table__header govuk-!-width-one-third">
+          Expires
+        </th>
+      </tr>
       </thead>
-      <tbody class="govuk-table__body govuk-!-font-size-14">
-        <tr class="govuk-table__row">
-          <td class="govuk-table__cell">
-            _ga
-          </td>
-          <td class="govuk-table__cell">
-            Used to distinguish anonymous users
-          </td>
-          <td class="govuk-table__cell">
-            2 years
-          </td>
-        </tr>
-        <tr>
-          <td class="govuk-table__cell">
-            _gid
-          </td>
-          <td class="govuk-table__cell">
-            Used to distinguish anonymous users
-          </td>
-          <td class="govuk-table__cell">
-            24 hours
-          </td>
-        </tr>
-        <tr>
-          <td class="govuk-table__cell">
-            _gat_gtag_<%= @cookie_ga_code %>
-          </td>
-          <td class="govuk-table__cell">
-            Throttle requests
-          </td>
-          <td class="govuk-table__cell">
-            10 minutes
-          </td>
-        </tr>
+      <tbody class="govuk-table__body">
+      <tr class="govuk-table__row">
+        <td class="govuk-table__cell">
+          _ga
+        </td>
+        <td class="govuk-table__cell">
+          Distinguish anonymous users
+        </td>
+        <td class="govuk-table__cell">
+          2 years
+        </td>
+      </tr>
+      <tr>
+        <td class="govuk-table__cell">
+          _gid
+        </td>
+        <td class="govuk-table__cell">
+          Distinguish anonymous users
+        </td>
+        <td class="govuk-table__cell">
+          24 hours
+        </td>
+      </tr>
+      <tr>
+        <td class="govuk-table__cell">
+          _gat_gtag_<%= @cookie_ga_code %>
+        </td>
+        <td class="govuk-table__cell">
+          Limit requests
+        </td>
+        <td class="govuk-table__cell">
+          10 minutes
+        </td>
+      </tr>
       </tbody>
     </table>
+
+    <p id="preferences" class="govuk-body">
+    <%= form_with model: @cookie_preferences, url: @cookie_settings_path, method: :post do |f| %>
+      <%= f.govuk_collection_radio_buttons :consent,
+        [%w[yes Yes], %w[no No]],
+        :first,
+        :last,
+        inline: true,
+        legend: { text: 'Do you want to accept Google Analytics cookies?', size: 's' } %>
+      <%= f.govuk_submit 'Save cookie settings' %>
+      <% end %>
+    </p>
```

Comment:
Do you need the `preferences` id? If so, can add it to the form.

```suggestion
    <%= form_with model: @cookie_preferences, url: @cookie_settings_path, method: :post do |f| %>
      <%= f.govuk_collection_radio_buttons :consent,
        [%w[yes Yes], %w[no No]],
        :first,
        :last,
        inline: true,
        legend: { text: 'Do you want to accept Google Analytics cookies?', size: 's' } %>
      <%= f.govuk_submit 'Save cookie settings' %>
    <% end %>
```

Reply:
👍